javascript/rexuws/app.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * 
+ * Rexuws (Replace Express with uWebsocket) is a framework based on uWebsocket.js
+ * It aims to be a drop-in, high speed replacement for express.
+ * 
+ * With uWebsockets.js clustering only properly works on Linux (not on WSL!).
+ * (presumably dependend the OS's of SO_REUSEPORT support)
+ * 
+ */
+
+
+const rex = require('rexuws').default;
+
+const app = rex({
+  useDefaultParser:true,
+})
+
+app.get("/", (req, res) => {
+  res.send("");
+})
+
+app.get("/user/:id", function (req, res) {
+  res.send(req.params.id);
+});
+
+app.post("/user", function (req, res) {
+  res.send("");
+});
+
+app.listen(3000, function () {
+  console.log("listening on port 3000");
```

Comment:
I think this could mess-up the results a little bit

Reply:
This only runs once after the socket has successfully been bound, not on every request.

So this should not cause any difference in req/s.

Reply:
Buuut... since it's not required I guess I'll remove it